File: android/src/main/kotlin/com/neverjp/background_task/lib/ChannelName.kt
package com.neverjp.background_task.lib

enum class ChannelName(
    val value: String
) {
    METHODS("com.neverjp.background_task/methods"),
    BG_EVENT("com.neverjp.background_task/bgEvent"),
    STATUS_EVENT("com.neverjp.background_task/statusEvent"),
}

File: ios/Classes/lib/ChannelName.swift
//
//  ChannelName.swift
//  background_task
//
//  Created by 中川祥平 on 2024/01/25.
//

import Foundation

enum ChannelName: String {
    case methods = "com.neverjp.background_task/methods"
    case bgEvent = "com.neverjp.background_task/bgEvent"
    case statusEvent = "com.neverjp.background_task/statusEvent"
    
    var value: String {
        rawValue
    }
}
